include/constants/global.h

```diff
@@ -8,6 +8,9 @@
 #include "config/pokemon.h"
 #include "config/overworld.h"
 #include "config/decap.h"
+#if TESTING
+#include "config/test.h"
+#endif
```

Comment:
Won't this do a redefinition warning thing because you're technically undefing them before they're defined?

Reply:
I think they should all be defined by this point, because they come from the other `config/*.h` includes above this line :thinking:

`#undef` will work regardless of whether they're defined, but yeah for sure you'll get duplicate definition errors if a later header tries to define them.

Reply:
Right, though `POKEMON_NAME_LENGTH` isn't

Reply:
Ah, damn! Good catch :) I'll move this to the end of the file